src/Apply.ts

```diff
@@ -13,13 +13,13 @@ export interface FantasyApply<F extends HKTS, A> extends FantasyFunctor<F, A> {
 export function applySecond<F extends HKT2S>(apply: StaticApply<F>): <L, A, B>(fa: HKT2<L, A>[F], fb: HKT2<L, B>[F]) => HKT2<L, B>[F]
 export function applySecond<F extends HKTS>(apply: StaticApply<F>): <A, B>(fa: HKT<A>[F], fb: HKT<B>[F]) => HKT<B>[F]
 export function applySecond<F extends HKTS>(apply: StaticApply<F>): <A, B>(fa: HKT<A>[F], fb: HKT<B>[F]) => HKT<B>[F] {
-  return <A, B>(fa: HKT<A>[F], fb: HKT<B>[F]) => apply.ap<B, B>(apply.map(() => identity, fa), fb)
+  return <A, B>(fa: HKT<A>[F], fb: HKT<B>[F]) => apply.ap<B, B>(apply.map<A, (x: B) => B>((_: A) => identity, fa), fb)
```

Comment:
@gcanti I am not sure if we should help or not the compiler (and curious reader) with type annotations (compiler, for speed).